sql/sql_expression_operations.cc

```diff
@@ -4197,6 +4197,52 @@ wildcards() const
 }
 
 
+  
+BoundSqlExpression
+GroupByKeyExpression::
+bind(SqlBindingScope & scope) const
+{
+    auto getGroupbyKey = scope.doGetGroupbyKey(index);
+
+    if (!getGroupbyKey.info) {
+        throw HttpReturnException(400, "scope " + MLDB::type_name(scope)
+                            + " doGetGroupbyKey '"
+                            + "' didn't return info");
+    }
+
+    auto outputInfo = getGroupbyKey.info->getConst(false);
```

Comment:
We need the (group by) row to evaluate its constValue(). Its a bit tricky, so I put a comment to show its not an oversight.